scripts/savers.py

```diff
@@ -249,6 +249,33 @@ def onResultsReady(self):
             output = { "data": self.results }
             fid.write(to_json(output))
 
+class TrackerDumpModeSaver(BaseSaver):
+    """ Callback functions used by tracker dump mode of pyxivo.
+    """
+    def __init__(self, args):
+        BaseSaver.__init__(self, args)
+
+    def onVisionUpdate(self, estimator, datum):
+        ts, content = datum
+
+        tracked_features = estimator.tracked_features()
+
+        for f in tracked_features:
+            id, kp, des = f
+
+            ts_and_id = np.array([ts, id]).reshape(1,2)
+            kp = np.reshape(kp, (1, 2))
+
+            feature_info = np.concatenate((ts_and_id, kp, des), axis=1)
+
+            with open(self.resultsPath, "a") as f:
+                np.savetxt(
```

Comment:
This line is currently printing everything out as a floating point number and it looks pretty crazy, even though all the data is correct. For example, here are two lines
```
7.687245611984000000e+12,1.001800000000000000e+04,2.410000000000000000e+02,3.520000000000000000e+02,2.440000000000000000e+02,1.170000000000000000e+02,2.180000000000000000e+02,2.090000000000000000e+02,2.490000000000000000e+02,1.580000000000000000e+02,6.100000000000000000e+01,1.450000000000000000e+02,2.130000000000000000e+02,6.100000000000000000e+01,2.240000000000000000e+02,6.200000000000000000e+01,2.080000000000000000e+02,2.430000000000000000e+02,1.220000000000000000e+02,2.150000000000000000e+02,5.700000000000000000e+01,3.100000000000000000e+01,8.000000000000000000e+01,1.610000000000000000e+02,1.860000000000000000e+02,2.090000000000000000e+02,1.820000000000000000e+02,1.780000000000000000e+02,2.000000000000000000e+00,6.400000000000000000e+01,1.840000000000000000e+02,2.300000000000000000e+02,1.440000000000000000e+02,2.390000000000000000e+02,1.600000000000000000e+01,1.490000000000000000e+02
7.687245611984000000e+12,1.001900000000000000e+04,2.080000000000000000e+02,3.550000000000000000e+02,1.610000000000000000e+02,1.900000000000000000e+02,5.900000000000000000e+01,2.350000000000000000e+02,6.600000000000000000e+01,9.600000000000000000e+01,1.580000000000000000e+02,9.600000000000000000e+01,5.800000000000000000e+01,1.150000000000000000e+02,1.800000000000000000e+01,1.970000000000000000e+02,1.980000000000000000e+02,1.260000000000000000e+02,2.070000000000000000e+02,2.460000000000000000e+02,4.400000000000000000e+01,6.900000000000000000e+01,2.290000000000000000e+02,2.400000000000000000e+01,8.500000000000000000e+01,7.000000000000000000e+01,6.600000000000000000e+01,2.290000000000000000e+02,4.400000000000000000e+01,1.710000000000000000e+02,1.820000000000000000e+02,5.300000000000000000e+01,1.500000000000000000e+01,1.470000000000000000e+02,1.490000000000000000e+02,6.800000000000000000e+01
7
```

List of changes I'm asking for:
1. print out timestamp as a floating point number, but in seconds instead of nanoseconds. (You'll see the other savers multiplying by 1e-9)
2. print out ID as a integer
3. If the descriptor is binary, don't print as floating point number. (To test, you can change the extracted descriptors from "BRIEF" to "SIFT" in the .cfg files.)